tests/test_registration.py

```diff
@@ -96,8 +96,7 @@ def test_signup_form_user_exists(customer_user):
     data = {'email': customer_user.email, 'password': 'password'}
     form = SignupForm(data)
     assert not form.is_valid()
-    error_message = 'User with this Email already exists.'
-    assert form.errors['email'] == [error_message]
+    assert form.errors['email']
```

Comment:
Doesn't matter to check if it was the expected error message? On my side because Travis was failing I made this change:
```patch
From bf9dc8ebe555984e83956b786d596f6a5022634a Mon Sep 17 00:00:00 2001
From: NyanKiyoshi <NyanKiyoshi@users.noreply.github.com>
Date: Thu, 1 Mar 2018 09:55:19 +0100
Subject: [PATCH] Should fix issues with Travis

---
 tests/test_registration.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_registration.py b/tests/test_registration.py
index b80e2cc1..d3487024 100755
--- a/tests/test_registration.py
+++ b/tests/test_registration.py
@@ -96,7 +96,7 @@ def test_signup_form_user_exists(customer_user):
     data = {'email': customer_user.email, 'password': 'password'}
     form = SignupForm(data)
     assert not form.is_valid()
-    error_message = 'User with this Email already exists.'
+    error_message = form.fields['email'].error_messages['unique']
     assert form.errors['email'] == [error_message]
```

You can ignore this message.